attributes/default.rb

```diff
@@ -1,2 +1,3 @@
 default['scrutinizer-whitelist']['source-url'] = 'https://scrutinizer-ci.com/api/meta'
 default['scrutinizer-whitelist']['priority'] = '05'
+default['scrutinizer-whitelist']['ports'] = [22]
```

Comment:
sorry, missinterpretted what I wrote, meaning more like

```
default['scrutinizer-whitelist']['ports'] = [node['sshd']['sshd_config']['Port']]
```

this by default will make it inherit from sshd cookbook's default